Foundation/GTMNSThread+Blocks.m

```diff
@@ -53,33 +53,13 @@ + (void)gtm_performBlockInBackground:(void (^)(void))block {
 
 #endif  // NS_BLOCKS_AVAILABLE
 
-@implementation GTMSimpleWorkerThread {
-  NSLock *sourceLock_;
-  CFRunLoopSourceRef source_;  // Protected by sourceLock_
-  CFRunLoopRef cfRunLoop_;
-}
-
-static void RunLoopContextEmptyFunc(void *info) {
-  // Empty because the source is used solely for signalling.
-  // The documentation for CFRunLoopSourceContext does not
-  // make it clear if you can have a null perform method.
-}
+@implementation GTMSimpleWorkerThread
 
 - (void)main {
   NSRunLoop *nsRunLoop = [NSRunLoop currentRunLoop];
-  {  // Braces are just to denote what is protected by sourceLock_
-    [sourceLock_ lock];
-    cfRunLoop_ = [nsRunLoop getCFRunLoop];
-    CFRetain(cfRunLoop_);
-    CFRunLoopSourceContext context = {0};
-    context.perform = RunLoopContextEmptyFunc;
-    source_ = CFRunLoopSourceCreate(NULL, 0, &context);
-    CFRunLoopAddSource(cfRunLoop_, source_, kCFRunLoopCommonModes);
-    [sourceLock_ unlock];
-  }
+  [nsRunLoop addPort:[NSPort port] forMode:NSDefaultRunLoopMode];
```

Comment:
Add a comment to document why a port needs to be added to the runloop.

Reply:
Done.